packages/keybr-textinput/lib/settings.ts

```diff
@@ -84,12 +90,18 @@ export const textDisplayProps = {
     WhitespaceStyle.Bullet,
   ),
   sounds: booleanProp("textDisplay.sounds", false),
+  enableSoundStyle: enumProp(
+    "textDisplay.enableSoundStyle",
+    EnableSoundStyle,
+    EnableSoundStyle.All,
+  ),
```

Comment:
Make these properties match the type definitions above.

`playSounds: enumProp("textDisplay.playSounds", PlaySounds, PlaySounds.None),`